src/components/Editor.vue

```diff
@@ -850,6 +850,8 @@ export default {
 				filePath: this.relativePath,
 				clientId: this.ydoc.clientID,
 				pendingStructs: this.ydoc.store.pendingStructs,
+				pendingStructsRemote:
+					this.$providers[0]?.remote.store.pendingStructs,
```

Comment:
`this.$provides[0]` was replaces with `this.syncProvider` in 12547cc05a3d8ce4900309db066ac7eddabcc35e, no?

Reply:
yes... will need to adjust after rebase on main.